sys/shell/Kconfig

```diff
@@ -11,3 +11,30 @@ menuconfig MODULE_SHELL
     depends on TEST_KCONFIG
 
 rsource "commands/Kconfig"
+
+menuconfig KCONFIG_MODULE_SHELL
+    bool "Configure the Shell interpreter"
+    depends on USEMODULE_SHELL
+
+if KCONFIG_MODULE_SHELL
+
+config SHELL_SHUTDOWN_ON_EXIT
+    bool "Shutdown RIOT on shell exit"
+    default y if CPU_NATIVE
+    help
+        Some systems (e.g Ubuntu 20.04) close stdin on CTRL-D / EOF
+        That means we can't just re-start the shell.
+        Instead terminate RIOT, which is also the behavior a user would expect
+        from a CLI application.
+
+config SHELL_NO_ECHO
+    bool "Disable echo"
+
+config SHELL_NO_PROMPT
+    bool "Disable prompt"
+
+config SHELL_PROMPT_CHAR_STR
+    string "Prompt symbol" if !SHELL_NO_PROMPT
+    default ">"
```

Comment:
I know there is no `char` type in Kconfig, but could you at least somehow restrict the length to one?

Reply:
I don't think I can from Kconfig, but I could try in the Makefile.include where the quotes are striped